project/src/main/java/com/google/sps/data/Places.java

```diff
@@ -58,7 +58,7 @@
         places.stream()
             .sorted(Comparator.comparing(Place::rating).reversed())
             .filter(place -> place.businessStatus() == BusinessStatus.OPERATIONAL)
-            .filter(place -> place.rating() >= minRating)
+            .filter(place -> Math.rint(place.rating()) >= minRating)
```

Comment:
So for a place with rating 3.8, the Math.rint result would be 4. 
Then, if `minRating == 4`, then this evaluates to `true`, although 3.8 < 4. 
Is this the intended behavior?

Reply:
Yes, that's the wanted behaviour (that's the purpose of the PR).
So for example, if someone specifies they want places with 5 starts, they'll get results (otherwise they probably won't get anything, because almost no place actually has 5 stars).

Reply:
Right, so I think in that case that `minRating` is a confusing name. Maybe rename it? 
Something like `approxMinRating` or similar.

Reply:
good point. done